### PR TITLE
Add upstream PR Review/Contribute Practices

### PR DESCRIPTION
**Pull request by chunfuwen**
This is follow-up for issue:#2748
Libvirt Team members summarize some rules or guidelines related to Upstream PR Review/Contribute as per past practices.
After reviewing, it is good to be merged and become common best practices in avocado-vt upstream.
If further feedbacks are available, related guidelines and rules can be constantsly evolved accordingly.

Signed-off-by: chunfuwen <chwen@redhat.com>

### CODE REVIEW

**Comment by chunfuwen** [comment]
This is a follow-up for issue :https://github.com/avocado-framework/avocado-vt/issues/2748

**Comment by pevogam** [comment]
I think this is best discussed after #2794 where we will reenable the vital parts of the CI and could add linters on top automating some of these requirements once properly considered.

**Comment by chunfuwen** [comment]
> I think this is best discussed after #2794 where we will reenable the vital parts of the CI and could add linters on top automating some of these requirements once properly considered.

Sorry, I am not sure how #2794 is linked with this PR.

**Comment by pevogam** [comment]
The main reason was that we have to see if everything works after allowing some minimal checks for the CI which also check styles and best practices. In fact I am confused - shouldn't the #2794 be closed now considering its PR is already merged?

**Comment by chunfuwen** [comment]
> The main reason was that we have to see if everything works after allowing some minimal checks for the CI which also check styles and best practices. In fact I am confused - shouldn't the #2794 be closed now considering its PR is already merged?

issue # 2794 was already addressed by merged PR:https://github.com/avocado-framework/avocado-vt/pull/2797 ,so I close it.

File: PR_Review_And_Contribute_Practices.txt
Comment (kylazhang) [inline PR_Review_And_Contribute_Practices.txt:21]: I think the description for No.15 is not so clear to understand

**Comment by dzhengfy** [comment]
@chunfuwen Could you help refine the commit message before we merge it? For example, remove below useless messages:
```
Signed-off-by: chunfuwen <chwen@redhat.com>
test
```

File: PR_Review_And_Contribute_Practices.txt
Comment (kylazhang) [inline PR_Review_And_Contribute_Practices.txt:49]: duplicate with No.6

File: PR_Review_And_Contribute_Practices.txt
Comment (chunfuwen) [reply PR_Review_And_Contribute_Practices.txt]: updated with :
Wait for request review for no more than 2 weeks on those PRs which have already 2 approvals

File: PR_Review_And_Contribute_Practices.txt
Comment (chunfuwen) [reply PR_Review_And_Contribute_Practices.txt]: merged with No.6

**Comment by chunfuwen** [comment]
> @chunfuwen Could you help refine the commit message before we merge it? For example, remove below useless messages:
> 
> ```
> Signed-off-by: chunfuwen <chwen@redhat.com>
> test
> ```

thanks, updated

**Review by dzhengfy — APPROVED** [review]
LGTM

**Review by kylazhang — APPROVED** [review]

**Pull request merged**
